snaps/features/custom-ui/index.md

```diff
@@ -176,7 +280,64 @@ module.exports.onHomePage = async () => {
 ```
 
 <p align="center">
-<img src={require("../assets/custom-ui-image.png").default} alt="Divider UI example" width="450px" style={{border: "1px solid #DCDCDC"}} />
+<img src={require("../../assets/custom-ui-image.png").default} alt="Divider UI example" width="450px" style={{border: "1px solid #DCDCDC"}} />
+</p>
+
+### `input`
+
+:::flaskOnly
+:::
+
+Outputs an input component for use in [interactive UI](interactive-ui.md).
+
+#### Parameters
+
+An object containing:
+
+- `name`: `string` - The name that will be used as a key to the event sent to
+  [`onUserInput`](../../reference/entry-points.md#onuserinput) when the containing form is submitted.
+- `inputType`: `string` - (Optional) Type of input.
+  Possible values are `text`, `number`, or `password`.
+  The default is `text`.
+- `placeholder`: `string` - (Optional) The text displayed when the input is empty.
+- `label`: `string` (Optional) The text displayed alongside the input to label it.
```

Comment:
I tried testing the `label` parameter but a label did not show up for me. The [example package](https://github.com/MetaMask/snaps/blob/main/packages/examples/packages/interactive-ui/src/index.ts) also doesn't seem to have a `label` example.

Reply:
cc. @GuillaumeRx

Reply:
The `label` property seems to still be in the code, I wonder if it's being used or not

https://github.com/MetaMask/snaps/blob/main/packages/snaps-sdk/src/ui/components/input.ts#L47

Reply:
This will be supported in 11.13, the docs are correct.